packages/devextreme/testing/tests/DevExpress.ui.widgets/loadPanel.tests.js

```diff
@@ -40,56 +41,193 @@ const LOADPANEL_MESSAGE_CLASS = 'dx-loadpanel-message';
 const MESSAGE_SELECTOR = '.' + LOADPANEL_MESSAGE_CLASS;
 const LOADPANEL_CONTENT_CLASS = 'dx-loadpanel-content';
 const LOADPANEL_PANE_HIDDEN_CLASS = 'dx-loadpanel-pane-hidden';
+const LOADINDICATOR_CLASS = '.dx-loadpanel-indicator';
+
+QUnit.module('accessibility', () => {
+    QUnit.module('message option tests', () => {
+        ['', undefined, null].forEach(state => {
+            const stateString = (state === '') ? 'empty' : state;
+            QUnit.test(`message is set to ${stateString} on init`, function(assert) {
+                const loadPanel = $('#loadPanel').dxLoadPanel({
+                    visible: true,
+                    message: state,
+                    showIndicator: false
+                }).dxLoadPanel('instance');
+                const loadPanelMessage = loadPanel.$wrapper().find('.dx-loadpanel-message').text();
 
-QUnit.module('init', {
-    beforeEach: function() {
-        fx.off = true;
-    },
+                assert.strictEqual(loadPanelMessage, '', 'Load Panel message value is empty');
```

Comment:
I see that we still have different understandings of what tests are needed :)
I asked you test that we have correct aria attributes after message option runtime change. That is testing of this [code row](https://github.com/DevExpress/DevExtreme/pull/27080/files#diff-53b011e4c36d90ad74e975c0d3719f7a2a69b4893caec03a31a2086859fed5bbR135). Now if I remove message || here tests are still green because u don't check aria-attributes after message option runtime check - instead of this you check dom element text equals passed message.

Reply:
[bb57796](https://github.com/DevExpress/DevExtreme/pull/27080/commits/bb57796b0f4e0953f491b0adee68e1f35762a715)